Database/Buffer/Backend/Archivist.ts

```diff
@@ -0,0 +1,75 @@
+import * as isoly from "isoly"
+import { Archive } from "../../../Database/Archive"
+import { Configuration } from "../../../Database/Configuration"
+import { Document } from "../../../Database/Document"
+import { KeyValueStore } from "../../../KeyValueStore"
+import { DurableObjectState, KVNamespace } from "../../../platform"
+import { Key } from "../../Key"
+import { Storage } from "./Storage"
+
+export class Archivist {
+	private constructor(
+		private readonly archive: Archive<Document & Record<string, any>>,
+		private readonly storage: Storage,
+		private readonly state: DurableObjectState,
+		private readonly partitions = ""
+	) {}
+	partition(...name: string[]) {
+		return new Archivist(
+			this.archive.partition(...name),
+			this.storage,
+			this.state,
+			this.partitions + name.join("/") + "/"
```

Comment:
`this.partitions` is never used and is also a duplicate variable of `this.archive.partitions` which then need to stay synced.